src/workerd/jsg/jsg.h

```diff
@@ -201,6 +230,26 @@ namespace workerd::jsg {
     registry.template registerMethod<NAME, decltype(&Self::name), &Self::name>();                  \
   } while (false)
 
+// Similar to JSG_METHOD, but also creates a V8 Fast API path for the method if its signature
+// is compatible with V8's Fast API Call conventions. This can significantly improve performance
+// for performance-critical methods with supported argument types.
+//
+// The same method will be used for both fast and slow paths, so it should be compatible with both.
+// See V8's v8-fast-api-calls.h for more details on Fast API requirements and limitations.
```

Comment:
Can we add a TODO comment here indicating that, long-term, the plan would be to try to merge `JSG_FAST_METHOD` into `JSG_METHOD` such that any compatible `JSG_METHOD` would just automatically be defined with a fast-mode alternative?

Reply:
I was planning on enabling it on this PR. No need to add a TODO to code since this PR already has a TODO in the pull-request description